File: 图文混排/TextKItDemo/TextKItDemo/ViewController.m
//
//  ViewController.m
//  TextKItDemo
//
//  Created by 胡明昊 on 17/4/13.
//  Copyright © 2017年 ccic. All rights reserved.
//

#import "ViewController.h"
#import "MHLabel.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    MHLabel *label = [[MHLabel alloc] initWithFrame:CGRectMake(100, 100, 200, 100)];
    [self.view addSubview:label];
    label.mhText = @"这是一段测试，如有疑问请点击http://www.baidu.com";
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 点击获取UIWebView中的图片/swift/HmhWebView/HmhWebView/ViewController.swift
//
//  ViewController.swift
//  HmhWebView
//
//  Created by 胡明昊 on 17/2/8.
//  Copyright © 2017年 ccic. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let webView = UIWebView(frame: view.frame)
        view.addSubview(webView)
        webView.loadRequest(URLRequest(url: URL(string: "https://www.baidu.com")!))
        webView.delegate = self
        
        webView.mh_didClickImageCall { (URLString) in
            print("图片地址 === \(URLString)")
        }
    }

}

extension ViewController: UIWebViewDelegate {
    
    
    
}
